util/realistic_data/ingest/s3_interface.py

```diff
@@ -34,15 +35,33 @@ def download_manifest_file(self) -> BytesIO:
         print("Downloaded channel manifest file from S3")
         return channel_manifest
 
-    def download_experiments(self) -> BytesIO:
+    def download_experiments(self) -> None:
         experiments_import = BytesIO()
         self.simulated_data_bucket.download_fileobj(
             Fileobj=experiments_import,
             Key="resources/experiments_for_mongoimport.json",
         )
         experiments_import.seek(0)
         print("Downloaded experiments (in format for mongoimport) from S3")
-        return experiments_import
+
+        binary_file = open(Config.config.database.remote_experiments_file_path, "wb")
+        binary_file.write(experiments_import.getvalue())
+        binary_file.close()
+
+        file_exists = False
+        while not file_exists:
+            file_exists = path.isfile(
+                Config.config.database.remote_experiments_file_path,
+            )
+            print(
+                f"{Config.config.database.remote_experiments_file_path} hasn't been"
+                " created yet..",
+            )
+            sleep(1)
+
+        print(
+            f"Written '{Config.config.database.remote_experiments_file_path}' to file",
+        )
```

Comment:
Don't have a problem with this, since we were doing it before in `LocalCommandRunner` so I'm sure there's a good reason, but I am curious as to what that reason is. How can the file not exist if we open, write and close it before reaching this loop? Is this just a general Python thing (allowing the the next lines to run before the file has actually been "saved")?